Scrumdinger/ScrumdingerApp.swift

```diff
@@ -11,7 +11,7 @@ import SwiftUI
 struct ScrumdingerApp: App {
     var body: some Scene {
         WindowGroup {
-            MeetingView()
+            ScrumsView(scrums: DailyScrum.sampleData)
         }
```

Comment:
☝️ This is the piece of code that defines the entry point of the app, the view that loads upon launching the app.